### PR TITLE
docs: add Realtime security docs

### PR DESCRIPTION
**Pull request by w3b6x9**
## What kind of change does this PR introduce?

docs update

## What is the new behavior?

Add/update documentation for Realtime Security (WALRUS)

### CODE REVIEW

File: web/docs/guides/auth/row-level-security.mdx [hunk 1]
```diff
@@ -270,14 +270,17 @@ using (
 
 ## Tips
 
-### Disable realtime for private tables
+### Enable Realtime for database tables
 
-Our realtime server doesn't provide per-user security. Until we build a more robust auth system for WebSockets, you can disable realtime functionality for any private tables. To do this, you can manage the underlying Postgres replication publication:
+Realtime server broadcasts database changes to authorized users depending on your Row Level Security (RLS) policies.
+We recommend that you create an entirely new publication and ensure that you enable row level security and set row security
```
Comment (kiwicopple) [inline web/docs/guides/auth/row-level-security.mdx:276]: > We recommend that you create an entirely new publication

Do we not provide a publication in the dashboard anymore?

Comment (w3b6x9) [reply web/docs/guides/auth/row-level-security.mdx:276]: @kiwicopple we do! let me remove `recommend that you create an entirely new publication` from a couple of places. i'll just say something like we recommend enabling rls and adding policies instead

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -97,10 +97,17 @@ services:
       DB_NAME: postgres
       DB_USER: postgres
```
Comment (kiwicopple) [inline docker/docker-compose.yml:98]: Should this one be `supabase_realtime_admin` now, or something similar?

Comment (w3b6x9) [reply docker/docker-compose.yml:98]: @kiwicopple we don't have a realtime admin role. `postgres` user works here as long as the db container runs as `postgres` user.

**Review by kiwicopple — APPROVED** [review]

**Pull request merged**